src/tasks/payload/payload_task.c

```diff
@@ -135,7 +135,39 @@ void beacon_down_command_test(slate_t *slate)
     }
 }
 
-void ping_command_test(slate_t *slate)
+bool heartbeat_check(slate_t *slate)
+{
+    const uint64_t TIME_THRESHOLD = 1000;
```

Comment:
Also put this as a #define statement at the top of the header file. `#define PAYLOAD_HEARTBEAT_TIMEOUT_MS 10000` <-- We can do say 10 second first?

Include the unit of time measurement in the variable name, it'll be helpful to avoid mismatching later on.